source/assets/stylesheets/locastyle/modules/_buttons.sass

```diff
@@ -118,6 +118,10 @@
       +single-transition(none)
       outline: none
 
+  .ls-form-inline &
+    [class*="ls-btn"] + [class*="ls-btn"]
+      margin-left: -1px
```

Comment:
This don't effect another types of buttons? If we put two normal buttons, they will have -1px of margin left?